airbyte-cdk/python/unit_tests/sources/file_based/scenarios/avro_scenarios.py

```diff
@@ -414,7 +414,7 @@
                     "col_date": "2022-05-29",
                     "col_time_millis": "06:00:00.456000",
                     "col_time_micros": "12:00:00.456789",
-                    "col_timestamp_millis": "2022-05-29T00:00:00.456000+00:00",
+                    "col_timestamp_millis": "2022-05-29T00:00:00.456+00:00",
```

Comment:
our previous "use default behavior" also led to it outputting not as milliseconds, but as microseconds. 

However @girarda i'm honestly a bit confused about `"col_time_millis": "06:00:00.456000",` being expected based on the code, since it's using the same timespec 🤔

Reply:
my understanding:
The parser returns a datetime equivalent to `datetime.datetime(2022,5,29,0,0,4,56000)`. The datetime isn't aware of the requested precision so it's just printed to the microsecond precision.

The updated parser turns it into a string with the right precision.